src/less/components.less

```diff
@@ -45,16 +45,16 @@
 @import 'components/pills';
 @import 'components/popovers';
 @import 'components/radios';
-@import 'components/reveals';
-@import 'components/search';
-@import 'components/selectors';
+@import 'components/reveal/index';
+@import 'components/search/index';
+@import 'components/selector/index';
 @import 'components/stepper'; // BETA
-@import 'components/tables';
-@import 'components/tabs';
+@import 'components/table/index';
+@import 'components/tab/index';
```

Comment:
sub-directory should be `src/less/components/tabset` to match `docs/components/tabset`

Reply:
Renamed `src/less/components/tab/*` -> `src/less/components/tabset/*`.